frontend/src/components/TournamentRegistrations/TournamentRegistrations.tsx

```diff
@@ -122,18 +124,37 @@ const PlayersList = (props: PlayersListProps) => {
   );
 };
 
-const PlayerNotFound = () => (
-  <Centered>
-    <Alert severity="info">
-      <AlertTitle>Nie znaleziono zawodnika?</AlertTitle>
-      Zapisz nowego poniżej.
-    </Alert>
-    <VerticalSpace height="1rem" />
-    <Button variant="contained" color="primary">
-      Dodaj i zapisz
-    </Button>
-  </Centered>
-);
+const PlayerNotFound = () => {
+  const [drawerState, setDrawerState] = useState<boolean>(false);
+
+  const toggleDrawer = (open: boolean) => () => {
+    setDrawerState(open);
+  };
+
+  return (
+    <Centered>
+      <Alert severity="info">
+        <AlertTitle>Nie znaleziono zawodnika?</AlertTitle>
+        Zapisz nowego poniżej.
+      </Alert>
+      <VerticalSpace height="1rem" />
+      <Button variant="contained" color="primary" onClick={toggleDrawer(true)}>
+        Dodaj i zapisz
+      </Button>
+      <Drawer
+        anchor={"bottom"}
+        open={drawerState}
+        onClose={toggleDrawer(false)}
+      >
+        <AddingPlayerForm
```

Comment:
Bardzo spoko, że tutaj umieściłeś ten formularz. Myślę, że kilknięcie buttona może być jako props :) I wtedy w tym miejscu będziemy od razu też zapisywać na turniej. A defaultowo będzie tylko dodanie nowego profilu.

Reply:
Ale ten formularz jest pod zapis na turniej :-P 

Kumam o co chodzi, żeby od razu zrobić ten sam formularz zarówno pod zapisywanie do turnieju jak i tworzenie nowego profilu, tylko nie rozumiem kliknięcie którego buttona  dać jako props (to chyba głupie pytanie, ale wolę się upewnić - tego `Dodaj i zapisz` )? I jak wgl dać  kliknięcie buttona jako props? Czy to tylko ma być tylko informacja dla AddingPlayerForm - czy to ma być utworzenie nowego profilu zawodnika, czy w sumie utworzenie nowego profilu i zapisanie do danego turnieju (wtedy jako props - tournamentId)?